library/general/src/modules/Report.rb

```diff
@@ -561,7 +569,9 @@ def LongWarning(warning_string, width: 60, height: 10)
       Builtins.y2warning(1, "%1", warning_string) if @log_warnings
 
       if @display_warnings
-        if Ops.greater_than(@timeout_warnings, 0)
+        if Mode.commandline
+          CommandLine.Print("Warning: #{error_string}")
+        elsif Ops.greater_than(@timeout_warnings, 0)
```

Comment:
thanks for fixing it. BTW it also shows how badly original API was designed by so many code duplications. Maybe something like Yast2::Popup should be done also for Report.

Reply:
Sure, this module deserves a bigger refactoring, but for SP2...